app/gui/controller/engine-protocol/src/language_server/types.rs

```diff
@@ -1175,10 +1175,11 @@ pub struct LibraryComponentGroup {
 ///
 /// For more information, see
 /// https://github.com/enso-org/design/blob/main/epics/basic-libraries/write-action-control/design.md.
-#[derive(Hash, Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Hash, Debug, Clone, Copy, PartialEq, Eq, Default, Serialize, Deserialize)]
```

Comment:
Clippy now requires `Default` derives, and it is a good thing for most places (except macros-generated code, urgh...)